.gitignore

```diff
@@ -3,3 +3,5 @@ build
 *.egg-info
 *.deb
 /dist
+.ropeproject
```

Comment:
Ok. Nice to me. I'll remove from my pr this. This happen because i use vim with jedi plugin and it create this folders.